Deployments/suex-enginfprojects-acpt/k8s-manifests/autoscale-app.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: webapp
```

Comment:
```suggestion
  name: enginfprojects
```

Reply:
webapp is very unspecific. The helm chart called `enginfprojects` suggest to call the hpa also `enginfprojects`.